_src/pattern-library/sass/components/_buttons.scss

```diff
@@ -93,6 +109,10 @@
     &.is-focused {
         background: palette(primary, base);
         color: palette(grayscale, white);
+
+        svg {
```

Comment:
All of these additions are making me realize that the button component visual attributes were never abstracted out into theming variables. @clrux, can you add a PL story to our backlog to do so?